packages/graphql/tests/integration/aggregations/where/edge/duration.int.test.ts

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { Driver, Session } from "neo4j-driver";
+import { graphql } from "graphql";
+import Neo4j from "../../../neo4j";
+import { Neo4jGraphQL } from "../../../../../src/classes";
+import { generateUniqueType } from "../../../../utils/graphql-types";
+
+describe("aggregations-where-edge-duration", () => {
+    let driver: Driver;
+    let neo4j: Neo4j;
+    let session: Session;
+    let typeDefs: string;
+
+    const typeMovie = generateUniqueType("Movie");
+    const typeActor = generateUniqueType("Actor");
+
+    let neoSchema: Neo4jGraphQL;
+
+    beforeAll(async () => {
+        neo4j = new Neo4j();
+        driver = await neo4j.getDriver();
+    });
+
+    beforeEach(async () => {
+        const typeDefs = `
+        type User {
```

Comment:
It looks like you're not using the unique types here and this could also do with another layer of indentation